src/util/prolog/prologSession.ts

```diff
@@ -4,7 +4,8 @@ import { format_answer, type } from "tau-prolog/modules/core";
 import getTopologicalOrder, {Edge} from "../getTopologicalOrder";
 import {fetchAllMatchesForAGroup} from "../regExpUtils";
 import { PrologResult } from "./prologResult";
-import { removeFileExtension, removeModuleDeclarations, removeNonFunctionalities } from "./prologStringManipulation";
+// PSM ~ prologStringManipulation
+import * as PSM from "./prologStringManipulation";
```

Comment:
I would prefer that you don't shorten a name that much. Imagine the file gets a lot larger and someone tests something, just to see that something in line 743 does not work. This person then proceeds trying to understand the code and reads "PSM". With this name that person then needs to jump around the file to look for that one comment you left.

Reply:
I think this is justified here though. Maybe `PSM` is not an ideal name, but I cannot think of anything that's better in every way.